lib/devise/passkeys/controllers.rb

```diff
@@ -9,6 +9,15 @@
 
 module Devise
   module Passkeys
+    # Passkeys Controllers encapsulate all methods that implement actions related to the Passkeys
+    #   The most common uses cases are: registration, authentication, and reauthentication
+    #   Other controllers and concerns should all inherit from here. As the following:
+    #     ReauthenticationChallenge concern: provide methods for reauthentication challenge
+    #     Reauthentication concern: provide methods for reauthentication
+    #     PasskeysControllerConcern: provide methods to manage passkeys
+    #     ReauthenticationControllerConcern: provide methods for handling reauthentication
+    #     RegistrationsControllerConcern: provide methods for handling registration
+    #     SessionsControllerConcern: provide methods for handing session
```

Comment:
@tcannonfodder : if I remember correctly, our conversation, was leading into having a summary of these here, but it seems duplicated with the more detailed documentation in the module itself.

But perhaps here and/or inside the module itself where we should indicate the difference between the concerns inside the `concerns` folder vs the others.
Ref: https://github.com/ruby-passkeys/devise-passkeys/issues/4#issuecomment-1590357907

Reply:
Yes, I agree!